README.md

```diff
@@ -154,6 +154,9 @@ The `wait_for_selector` option can also be used to wait until an element appears
 
 The `wait_for_timeout` option can also be used to wait the specified number of milliseconds have elapsed.
 
+The `request_failure` option, when enabled, will raise a `Grover::JavaScript::RequestFailedError`
```

Comment:
hmm the more I think about it the more this option name doesn't really fit. How about `raise_request_failures` or `raise_on_request_failure` ? Just tells the person reading the config exactly what to expect when enabled. Cause you know.. no one actually WANTS a request failure right?? ie `request_failure: true` would seem to suggest it will always fail :)

Reply:
I like `raise_on_request_failure`. I was trying to match the puppeteer function naming, but I think that your suggestion is better.